clang/tools/clang-check/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ set( LLVM_LINK_COMPONENTS
   ${LLVM_TARGETS_TO_BUILD}
   Option
   Support
+  MCCAS
```

Comment:
This looks suspicious because neither MC or CAS are listed. Why is this needed?

Reply:
Yep, I was able to remove that one